File: app/src/main/java/com/comfortspace/data/model/Notification.kt
package com.comfortspace.data.model

import com.google.gson.annotations.SerializedName

data class Notification(
    val notificationId: Int,
    val title: String,
    val message: String,
    val type: String,
    @SerializedName("created_at")
    val createdAt: String,
    val isRead: Boolean = false
)

File: lab4/src/Models/Notification.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ComfortSpace.Models
{
    [Table("Notifications")]
    public class Notification
    {
        [Key]
        [Column("notification_id")]
        public long NotificationId { get; set; }

        [Column("user_id")]
        public int UserId { get; set; }

        [Column("title")]
        public string Title { get; set; }

        [Column("message")]
        public string Message { get; set; }

        [Column("type")]
        public string Type { get; set; }

        [Column("created_at")]
        public DateTime CreatedAt { get; set; }

        // Navigation
        [ForeignKey("UserId")]
        public User User { get; set; }
    }
}